src/ctree-app/ctree-landing.html

```diff
@@ -79,44 +79,70 @@
         --app-secondary-color: black;
       }
       #mainLayout app-toolbar {
-        padding: 0 8px 0 0;
+        padding: 15px 10px 10px 15px;
+        @apply --layout-horizontal;
+        @apply --layout-end-justified;
+        background-color: rgba(255, 255, 255, 0.95);
+
       }
       #content {
         @apply --layout-fit;
         overflow-y: auto;
         -webkit-overflow-scrolling: touch;
       }
-      #headerTitle, #menuButton {
-        display: none;
+      #headerTitle {
+        color: #63A375;
+        font-size: 23px;
+
+      }
+      #menuButton{
+        color: black;
+        float: right;
       }
       app-drawer ul {
         text-align: left !important;
       }
       app-header-layout {
-        height: 100vh;
+        height: 200vh;
```

Comment:
Hmm, I'll need to check this out in practice, but it looks strange to essentially be 200% of the view height.  Maybe I'm wrong, but I'll check this out when testing.